commerce-product-options-web/src/main/java/com/liferay/commerce/product/options/web/internal/display/context/BaseCPOptionsDisplayContext.java

```diff
@@ -230,11 +230,11 @@ protected void setDefaultOrderByType(String defaultOrderByType) {
 	}
 
 	protected final ActionHelper actionHelper;
-	protected final CPRequestHelper cpRequestHelper;
 	protected final HttpServletRequest httpServletRequest;
 	protected final LiferayPortletRequest liferayPortletRequest;
 	protected final LiferayPortletResponse liferayPortletResponse;
 	protected final PortalPreferences portalPreferences;
+	protected final CPOptionsRequestHelper requestHelper;
```

Comment:
`CPOptionsRequestHelper requestHelper` should be `CPOptionsRequestHelper cpOptionsRequestHelper`